src/app/pages/CpsMap/index.test.jsx

```diff
@@ -245,8 +247,32 @@ describe('CPS MAP Page', () => {
   it('should render lastPublished timestamp for Pidgin', () => {
     expect(getByText('New Informate 20 November 2019')).toBeInTheDocument();
   });
+
   it('has a single "main" element, and a single "complementary" element (a11y)', async () => {
     expect(document.querySelectorAll(`[role='main']`).length).toBe(1);
     expect(document.querySelectorAll(`[role='complementary']`).length).toBe(1);
   });
 });
+
+it('should not show the pop-out timestamp when allowDateStamp is false', async () => {
+  const pageDataWithHiddenTimestamp = assocPath(
+    ['metadata', 'options', 'allowDateStamp'],
+    false,
+    await preprocessor(mapPageData, cpsAssetPreprocessorRules),
+  );
+
+  const { asFragment } = render(
+    createAssetPage({ pageData: pageDataWithHiddenTimestamp }, 'pidgin'),
+  );
+
+  expect(document.querySelector('main time')).toBeNull();
+  expect(asFragment()).toMatchSnapshot();
```

Comment:
Will we remove this - omit snapshot tests?

Reply:
aye one snapshot per page makes sense to me, unless there's something at runtime (eg, different data) that can cause widely divergent DOM